api/sixtyfps-cpp/include/sixtyfps.h

```diff
@@ -477,7 +477,7 @@ class VectorModel : public Model<ModelData>
 template<typename C, typename ModelData>
 class Repeater
```

Comment:
I guess the Repeater itself need to be moved.

Reply:
Done, along with more types.